addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3948,6 +3948,9 @@ registry.BackgroundImage = SnippetOptionWidget.extend({
         } else {
             this.$target.css('background-image', '');
             this.$target.removeClass('oe_img_bg');
+            delete this.$target[0].dataset.mimetypeBeforeConversion;
+            delete this.$target[0].dataset.bgSrc;
```

Comment:
Seeing that other data attributes are removed correctly (`data-original-id`, `data-original-src`, etc.), I looked for where they were removed and there seems to be a list defined in `image_processing.js` ... => 
```js
const modifierFields = [
    'filter',
    'quality',
    'mimetype',
    'glFilter',
    'originalId',
    'originalSrc',
    'resizeWidth',
    'aspectRatio',
];
```
Could you take a look

Reply:
I've updated the `modifierFields` according to your comment. That seems nicer. Thanks.